lang/tr/lang.php

```diff
@@ -7,26 +7,32 @@
     ],
     'blog' => [
         'menu_label' => 'Blog',
-        'menu_description' => 'Gönderileri Yönet',
+        'menu_description' => 'Blog Gönderilerini Yönet',
         'posts' => 'Gönderiler',
-        'create_post' => 'blog gönderisi',
+        'create_post' => 'Blog gönderisi',
         'categories' => 'Kategoriler',
-        'create_category' => 'blog kategorisi',
+        'create_category' => 'Blog kategorisi',
         'tab' => 'Blog',
         'access_posts' => 'Gönderileri yönetebilsin',
         'access_categories' => 'Blog kategorilerini yönetebilsin',
         'access_other_posts' => 'Diğer kullanıcıların gönderilerini yönetebilsin',
         'access_import_export' => 'Gönderileri içeri/dışarı aktarabilsin',
         'access_publish' => 'Gönderi yayınlayabilsin',
+        'manage_settings' => 'Blog ayarlarını yönet',
         'delete_confirm' => 'Emin misiniz?',
         'chart_published' => 'Yayınlandı',
         'chart_drafts' => 'Taslaklar',
         'chart_total' => 'Toplam',
+        'settings_description' => 'Blog ayarlarını yönet',
+        'show_all_posts_label' => 'Tüm gönderileri yönetim paneli kullanıcılarına göster',
+        'show_all_posts_comment' => 'Hem yayınlanmış hem de yayınlanmamış gönderileri önyüzde yönetim paneli kullanıcılarına göster.',
+        'tab_general' => 'Genel'
+        
```

Comment:
Remove this extra line of whitespace please

Reply:
Done. I didn't removed it at the beginning to match with `en/lang.php`. Now I also removed that line of space from `en/lang.php` too.